### PR TITLE
[New Rule] Unusual File Creation - Alternate Data Stream

### LINKED ISSUE #862: [New Rule] Unusual Alternate Data Stream File Creation
## Description
Identifies suspicious creation of Alternate Data Streams on highly targeted files. This is uncommon for legitimate processes and sometimes done by adversaries to hide malware.

## Required Info

- **Eventing Sources:**
`winlogbeat-*`, `logs-endpoint.events.*`

- **Platforms**
`windows`

## Optional Info
- **References:**

## Syntax
```
file where event.type:"creation" and 
  file.name:"*:*" and not
  file.name:"*:zone.identifier" and
  file.extension:(
    "pdf", 
    "dll", 
    "png", 
    "exe", 
    "dat", 
    "com", 
    "bat", 
    "cmd", 
    "sys")
```

## Example Data
image

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #862 

## Summary
Identifies suspicious creation of Alternate Data Streams on highly targeted files. This is uncommon for legitimate files and sometimes done by adversaries to hide malware.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
we already have a [rule ](https://github.com/elastic/detection-rules/blob/main/rules/windows/defense_evasion_unusual_dir_ads.toml)that looks for the execution-time.

**Review by threat-punter — APPROVED** [review]
If the team is confident that the leading wildcards won't impact performance, then LGTM 👍

**Comment by bm11100** [comment]
> we already have a [rule ](https://github.com/elastic/detection-rules/blob/main/rules/windows/defense_evasion_unusual_dir_ads.toml)that looks for the execution-time.

One is looking at process execution events while this one is looking for unusual file creations. I think we move forward with both.

**Review by Samirbous — APPROVED** [review]

**Review by devonakerr — APPROVED** [review]

File: rules/windows/defense_evasion_unusual_ads_file_creation.toml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+name = "Unusual File Creation - Alternate Data Stream"
+risk_score = 47
+rule_id = "71bccb61-e19b-452f-b104-79a60e546a95"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+file where event.type : "creation" and
+  file.name : "*:*" and
```
Comment (brokensound77) [inline rules/windows/defense_evasion_unusual_ads_file_creation.toml:25]: what if we changed this to the following, to eliminate the leading wildcard

```suggestion
  file.path : "C:\\*:*" and
```

File: rules/windows/defense_evasion_unusual_ads_file_creation.toml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+license = "Elastic License"
+name = "Unusual File Creation - Alternate Data Stream"
+risk_score = 47
+rule_id = "71bccb61-e19b-452f-b104-79a60e546a95"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+file where event.type : "creation" and
```
Comment (brokensound77) [inline rules/windows/defense_evasion_unusual_ads_file_creation.toml:24]:
```suggestion
file where event.type == "creation" and
```

File: rules/windows/defense_evasion_unusual_ads_file_creation.toml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  file.name : "*:*" and
+  not file.name : "*:zone.identifier*" and
```
Comment (brokensound77) [inline rules/windows/defense_evasion_unusual_ads_file_creation.toml:26]:
```suggestion
  not file.path : "C:\\*:zone.identifier*" and
```

File: rules/windows/defense_evasion_unusual_ads_file_creation.toml (hunk 1 above)
Comment (bm11100) [reply rules/windows/defense_evasion_unusual_ads_file_creation.toml:25]: that's a good suggestion, and tested successfully, will add

**Pull request merged**